.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create a report to help us fix or improve an issue
+---
+
+## Describe the bug
+
+_A clear and concise description of what the bug is, and the conditions when it occurs._
+
+## How to reproduce
+
+_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._
+
+Steps to reproduce the behavior:
+
+1.  Go to '...'
+2.  Click on '....'
+3.  Scroll down to '....'
+4.  See error
+
+## Expected behavior
+
+_A clear and concise description of what you expected to happen._
+
+## Versions
+
+**Clarity project:**
+
+- [ ] Clarity Core
```

Comment:
Now we don't have to specify Clarity Core vs Clarity Angular, so I think this section should go away.

Reply:
Yeah I had that thought too - the Core repo still has this in their templates and I wasn't sure if devs might struggle with which repo they're supposed to be in?